cf_units/tests/integration/parse/test_parse.py

```diff
@@ -242,7 +242,7 @@ def test_known_issues(_, unit_str, expected):
     # These are the cases that don't work yet but which do work with udunits.
 
     # Make sure udunits can read it.
-    cf_units.Unit(unit_str).symbol
+    cf_units.Unit(unit_str).symbol  # noqa: B018
```

Comment:
Perhaps re-write as ``_ = ...`` instead of the noqa

Reply:
Done